source/settings/command-line-options.txt

```diff
@@ -116,6 +116,15 @@ value in the :guilabel:`Settings` panel.
        
        To learn more, see :ref:`compass-enable-dev-tools`.
 
+   * - .. option:: --enableGenAIFeatures
+     - Allow |compass-short| to use generative AI for natural language querying.
+       This feature requires an Atlas login and a deployed Atlas cluster.
+
+   * - .. option:: --enableGenAIDocumentPassing
```

Comment:
There's a typo here; the option is actually `--enableGenAISampleDocumentPassing`

Reply:
Fixed